### DISCUSSION TITLE
How to upload more than 5000 files?

### DISCUSSION THREAD

**Discussion by doggy8088** [vercel/vercel#6617] [General] [answered]
I have 6694 files in my folder. When I use `vercel` to upload files, I get `Error! Too many requests - try again in 24 hours (more than 5000, code: "api-upload-free").`. Is there any way I can upload my static web site to Vercel?

**Comment by leerob** [comment, marked as answer]
As you mentioned, you're hitting the [free tier limit](https://vercel.com/docs/platform/limits#deployments-per-day-(free)).
<img width="718" alt="CleanShot 2021-08-21 at 15 23 44@2x" src="https://user-images.githubusercontent.com/9113740/130333990-fd24c54f-44af-4509-80d2-656deea6d1b5.png">

If you would like to upload more than 5,000 files, you can update to a Pro team - or use a service like AWS S3 for static storage 👍

**Reply by doggy8088** [reply]
Hi @leerob , Can I upload a zip file and unzip on Vercel automatically?

**Reply by leerob** [reply]
You can host a `.zip` file, yes, but it won't unzip automatically when you access/download it.
